springboot/src/main/resources/logback.xml

```diff
@@ -6,5 +6,6 @@
   </root>
   <logger name="org.springframework.web" level="ERROR"/>
   <logger name="org.kie.hacep" level="INFO"/>
+  <logger name="org.drools.compiler.kie.builder" level="DEBUG"/>
```

Comment:
Is for the reviewers :) , is the only way to see the update of the kjar in the logs

Reply:
I agree with @jiripetrlik because this file will be included in the jar. If it is for testing purpose please move it to a test resouces file or similar